cmd/juju/cloud/show_test.go

```diff
@@ -55,24 +85,26 @@ clouds:
         endpoint: http://london/1.0
     config:
       bootstrap-timeout: 1800
```

Comment:
Line 87 is the config.yaml file used for show-cloud, so it should be config.  At line 107, bootstrap-timeout is shown in as part of bootstrap-config.

What do we expect the region-config output to look like with these changes?
region-config:
  london:
    bootstrap-config:
      bootstrap-timeout: 1800
    model-config:
      use-floating-ip: true